catroid/src/androidTest/java/org/catrobat/catroid/uiespresso/content/brick/rtl/RtlBrickTest.java

```diff
@@ -693,7 +691,8 @@ public void userBricks() {
 		assertEquals(arLocale.getDisplayLanguage(), Locale.getDefault().getDisplayLanguage());
 		assertTrue(RtlUiTestUtils.checkTextDirectionIsRtl(Locale.getDefault().getDisplayName()));
 		openCategory(R.string.category_user_bricks);
-		onView(allOf(withId(android.R.id.list), withParent(withId(R.id.fragment_user_defined_brick_list)))).check(matches(hasChildCount(0)));
+
+		checkIfBrickISRtl(ReportBrick.class, R.id.brick_report_layout);
```

Comment:
Tests fail with
```
06:17:13  /home/catroid/jenkins_slave/workspace/Catroid_PR-3990/catroid/src/androidTest/java/org/catrobat/catroid/uiespresso/content/brick/rtl/RtlBrickTest.java:695: error: cannot find symbol
06:17:13  		checkIfBrickISRtl(ReportBrick.class, R.id.brick_report_layout);
06:17:13  		                  ^
06:17:13    symbol:   class ReportBrick
06:17:13    location: class RtlBrickTest
```
see https://jenkins.catrob.at/job/Catroid/job/PR-3990/16/execution/node/79/log/

Probably due to https://github.com/Catrobat/Catroid/blob/5f6f631fa48bca33e78f02a8152585c52a9d6437/catroid/src/androidTest/java/org/catrobat/catroid/uiespresso/content/brick/rtl/RtlBrickTest.java#L248
i.e., you need the resource also in the arabic strings.xml resources.

Reply:
Should be working now